utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { User } from '../models/models';
+
+export const transformUser = (user: any): User => ({
+  userId: user.id,
+  name: user.name,
+  email: user.email,
+  postId: '',
```

Comment:
This is to transform the users' posts, not a user. Also, I don't see this being called anywhere in the code?

Reply:
getUserPosts
  
  it's called from the controller.ts method
  
  `  async getUserPosts(userId: string): Promise<TransformedPosts[]> {
    try {
      const posts = await getPostsByUserId(userId);
      const transformedPosts = usersUtil.transformUserPosts(posts);

      return transformedPosts;
    } catch (error) {
      console.error('Error calling API:', error);
      throw error;
    }
  }`